src/Cmfd.cpp

```diff
@@ -959,12 +962,12 @@ void Cmfd::updateMOCFlux() {
           iter != _cell_fsrs.at(i).end(); ++iter) {
 
           /* Set new flux in FSR */
-            _FSR_fluxes[*iter*_num_moc_groups+h] = getFluxRatio(i,h)
+          _FSR_fluxes[*iter*_num_moc_groups+h] = getUpdateRatio(i,h, *iter)
```

Comment:
The `updateMOCFlux` method updates the FSR flux using the old and new CMFD flux. In short, it sets the new FSR flux to the old fsr flux times some ratio of the new-to-old CMFD flux in the cell containing and near the FSR. I'll change the comment to:

`/* Update FSR flux using ratio of new to old CMFD flux */`

If you think of something better, just let me know.